deploy/etc/crds/ftns.crd.yaml

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020 The Authors.
+
+# Authors: Phu Tran          <@phudtran>
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: ftns.zeta.com
+spec:
+  scope: Namespaced
+  group: zeta.com
+  versions:
+    - name: v1
+      served: true
+      storage: true
+  names:
+    kind: Ftn
+    plural: ftns
+    singular: ftn
+    shortNames:
+      - ftn
+      - ftns
+  additionalPrinterColumns:
+    - name: Dft
+      type: string
+      priority: 0
+      JSONPath: .spec.dft
+      description: The DFT that this FTN belongs to
+    - name: Droplet
+      type: string
+      priority: 0
+      JSONPath: .spec.droplet
+      description: The droplet where this ftn is deployed
+    - name: Nextnode
+      type: string
+      priority: 0
+      JSONPath: .spec.nextnode
+      description: The next replica following this ftn
+    - name: Parentchain
+      type: string
+      priority: 0
+      JSONPath: .spec.parentchain
+      description: The parent chain of this ftn
+    - name: Augmentedchain
+      type: string
+      priority: 0
+      JSONPath: .spec.augmentedchain
+      description: The chain this ftn is augmented onto
+    - name: Status
+      type: string
+      priority: 0
+      JSONPath: .spec.status
```

Comment:
Since Head and tail FTN has special functionality, shall we add flags to indicate their role in the chain?

Reply:
Synced offline.
Will add these fields if they're needed in the future.